routing/dht/routing.go

```diff
@@ -40,19 +40,24 @@ func (dht *IpfsDHT) PutValue(ctx context.Context, key u.Key, value []byte) error
 		return err
 	}
 
-	peers := dht.routingTable.NearestPeers(kb.ConvertKey(key), KValue)
-
-	query := newQuery(key, dht.network, func(ctx context.Context, p peer.ID) (*dhtQueryResult, error) {
```

Comment:
I definitely agree the DHT needs a good refactor, but i think thats lower priority than getting everything else working nicely. (and getting all the tests passing once more!)